plugins/fabric8-multi-tenant-manager/src/main/java/com/redhat/che/multitenant/TenantDataProvider.java

```diff
@@ -179,6 +118,72 @@ private String getResponseBody(final String endpoint, final String keycloakToken
     return response.asString();
   }
 
+  private class TenantDataLoader extends CacheLoader<CacheKey, UserCheTenantData> {
+
+    @Override
+    public UserCheTenantData load(CacheKey cacheKey) throws InfrastructureException {
+      if (standalone) {
```

Comment:
Do we need this standalone part at all? AFAIK we wanted to remove single user functionality completely. Maybe it worth removing this part now?

Reply:
AFAIK, standalone = minishift rh-che addon

Reply:
oh, I forgot about this use-case,thanks